bin/language.txt

```diff
@@ -817,7 +817,7 @@ Monarchy
 Republic
 Fundamentalism
 Communism
-Democracy
+Capitalism
```

Comment:
This change is not reflected in all documentation.. I'm also not actually sure about the strong desire for that change to happen..